src/handlers/rendered_link.rs

```diff
@@ -47,6 +47,10 @@ async fn add_rendered_link(
                     .trigger_files
                     .iter()
                     .any(|tf| f.filename.starts_with(tf))
+                    && !config
+                        .exclude_files
+                        .iter()
+                        .any(|tf| f.filename.starts_with(tf))
```

Comment:
This won't work, because the filename contains the whole path from root (try `gh api repos/rust-lang/rust-forge/pulls/918/files`). Maybe we could check only the filename itself here, not the whole path?

Probably it would be also more accurate to rename `trigger_files` to `trigger_paths`...

Reply:
Well, it's on purpose. `exclude_files` is the mirror of `trigger_paths` which works on the filepath.

Reply:
Yeah, I know. It's just that the `SUMMARY.md` default isn't very useful here.

Reply:
Is the intention to put index .md files into `exclude_files`? What happens if a patch to the rust-forge repository only changes the file `SUMMARY.md` and that file is in the exclude list? That might be a corner case.

maybe put `exclude_files` back in if the filtering returned nothing? :thinking: